R/overline.R

```diff
@@ -108,55 +108,64 @@ lineLabels <- function(sl, attrib) {
 
 #' Convert series of overlapping lines into a route network
 #'
-#' This function takes a series of overlapping lines
-#' and converts them into a single route network.
-#'
-#' The function can be used to estimate the amount of transport 'flow'
-#' at the route segment level based on input datasets from routing services,
-#' for example linestring geometries created with the `route()` function.
-#'
-#' @param attrib A character vector corresponding to the variables in
-#' `sl$` on which the function(s) will operate.
-#' @param fun The function(s) used to aggregate the grouped values (default: sum).
-#' If length of `fun` is smaller than `attrib` then the functions are
-#' repeated for subsequent attributes.
-#' @param na.zero Sets whether aggregated values with a value of zero are removed.
+#' This function takes a series of overlapping lines and converts them into a
+#' single route network.
+#'
+#' The function can be used to estimate the amount of transport 'flow' at the
+#' route segment level based on input datasets from routing services, for
+#' example linestring geometries created with the `route()` function.
+#'
+#' @param attrib A character vector corresponding to the variables in `sl$` on
+#'   which the function(s) will operate.
+#' @param fun The function(s) used to aggregate the grouped values (default:
+#'   sum). If length of `fun` is smaller than `attrib` then the functions are
+#'   repeated for subsequent attributes.
+#' @param na.zero Sets whether aggregated values with a value of zero are
+#'   removed.
 #' @param ... Arguments passed to `overline2`
 #' @inheritParams gsection
 #' @inheritParams overline2
 #' @author Barry Rowlingson
-#' @references
-#' Rowlingson, B (2015). Overlaying lines and aggregating their values for
-#'  overlapping segments. Reproducible question from
-#'  <https://gis.stackexchange.com>. See <https://gis.stackexchange.com/questions/139681/overlaying-lines-and-aggregating-their-values-for-overlapping-segments>.
+#' @references Morgan, M (2020). Travel flow aggregation: Nationally scalable
```

Comment:
Great to reference to paper but I think it's as follows: 

Morgan M, Lovelace R. Travel flow aggregation: Nationally scalable methods for interactive and online visualisation of transport behaviour at the road network level. Environment and Planning B: Urban Analytics and City Science. July 2020. doi:10.1177/2399808320942779

That's from https://journals.sagepub.com/doi/full/10.1177/2399808320942779#articleCitationDownloadContainer